baseplate/clients/thrift.py

```diff
@@ -166,6 +167,15 @@ def _call_thrift_method(self: Any, *args: Any, **kwargs: Any) -> Any:
                     if span.flags:
                         prot.trans.set_header(b"Flags", str(span.flags).encode())
 
+                    min_timeout = value or None
```

Comment:
the `or None` part is actually unnecessary since the value emitted from retry_policu is either time remaining or none, so here it can be just `min_timeout = value` (or `time_remaining` if you renamed it as suggested above)